DESCRIPTION

```diff
@@ -15,7 +15,10 @@ Description: The Predictive Ecosystem Carbon Analyzer (PEcAn) is a scientific
     to the PEcAn API server & interact with it.
 Imports:
   httr,
-  jsonlite
+  jsonlite,
+  grid,
```

Comment:
if we remove the display function, we don't need the png and grid libraries, which minimizes the list of dependencies. People using this code can put it in their own set of dependencies.